src/components/footer/footer.jsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import './footer.scss';
+import PvdGeeksLogo from '../pvd-geeks-logo/pvd-geeks-logo';
+import FacebookIcon from '../../components/facebook-icon/facebook-icon';
+import SlackIcon from '../../components/slack-icon/slack-icon';
+import TwitterIcon from '../../components/twitter-icon/twitter-icon';
+
+export default class Footer extends React.Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    return (
+      <div className="footer row">
+        <div className="footer-top d-flex">
+          <div className="col-md-2">
+            <div className="logo">
+              <a href="/"><PvdGeeksLogo /></a>
+            </div>
+          </div>
+
+          <div className="social-container col-md-8 offset-md-2 d-flex justify-content-end align-items-center">
```

Comment:
I wonder if this makes the case for a reusable component for these social network links?  Something _social-networking-links_